packages/address/src/address.ts

```diff
@@ -98,7 +100,7 @@ export default class Address extends AbstractAddress {
    *
    * @returns The `bech32Address` property as an {@link EvmAddress | `EvmAddress`}
    */
-  toEvmAddress(): EvmAddress {
+  toWrappedEvmAddress(): EvmAddress {
```

Comment:
I'm unsure about the name change. Is there a reason to change it?

Reply:
I was trying to create some distinction between an Evm Address, being the unwrapped 20 byte address that `fromEvmAddress` takes as a function parameter, and the wrapped 256 bit Evm Address Sway type. 🤔

Reply:
That's a good point, I'll remove the infix